schemas/x5f2_status.fbs

```diff
@@ -0,0 +1,15 @@
+// Status message
+// To act as heatbeat and status report from any software
+
+file_identifier "x5f2";
+
+table Status {
+    software_name : string;      // Name of the software publishing the status message
+    software_version : string;   // Version number of the software publishing the status message
+    system_name : string;        // System name to identify where the software is running
```

Comment:
With "system_name", do you mean which operating system or which host? If it is the former, I see that as probably not necessary and you could/should replace it with a host name string. If it is the latter, please rename `system_name` to (e.g.) `host_name`.

Reply:
Yes I meant host.